crypto/ocsp/ocsp_client.c

```diff
@@ -3,22 +3,113 @@
 
 #include "internal.h"
 
-OCSP_ONEREQ *OCSP_request_add0_id(OCSP_REQUEST *req, OCSP_CERTID *cid)
-{
-    OCSP_ONEREQ *one = OCSP_ONEREQ_new();
-    if (one == NULL) {
-      return NULL;
+OCSP_ONEREQ *OCSP_request_add0_id(OCSP_REQUEST *req, OCSP_CERTID *cid) {
```

Comment:
Do we have existing tests for these get/set/add functions?

Reply:
1. For `OCSP_request_add0_id` we already have a test [here](https://github.com/aws/aws-lc/blob/ocsp-request/crypto/ocsp/ocsp_test.cc#L809-L840) in the branch.
2. As for `OCSP_request_set1_name` and `OCSP_request_add1_cert`, I assumed we already had coverage for  since `OCSP_request_sign` relies on them succeeding. I could add a more targeted test for them if needed?